profile.md

```diff
@@ -459,15 +459,13 @@ For a given storage resource, the defined authorizations include:
 
 
 
-*   **storage.read**: Read data.  Only applies to 'online' resources such as disk (as opposed to 'nearline' such as tape where the **stage** authorization should be used in addition).
+*   **storage.read**: Read data.  Only applies to 'online' resources such as disk, as opposed to 'nearline' resources such as tape, where the **stage** authorization should be used instead.
 *   **storage.create**: Upload data.  This includes renaming files if the destination file does not already exist. This capability includes the creation of directories and subdirectories at the specified path, and the creation of any non-existent directories required to create the path itself. This authorization does **not** permit overwriting or deletion of stored data.  The driving use case for a separate `storage.create` scope is to enable the stage-out of data from jobs on a worker node.
 *   **storage.modify**: Change data.  This includes renaming files, creating new files, and writing data.  This permission includes overwriting or replacing stored data in addition to deleting or truncating data.  This is a strict superset of `storage.create`.
-*   **storage.stage**: Read the data, potentially causing data to be staged from a nearline resource to an online resource. This is a superset of `storage.read`.
+*   **storage.stage**: Stage and/or read data. This scope causes data to be staged, when needed, from a nearline resource to an online resource, to allow the data to be read. Because staging is always done in order to read the given data next, the `storage.stage` scope has been made a superset of `storage.read`. This allows the same token to be used for both operations, while avoiding the need to include the `storage.read` scope for every given path in addition. Note: storage services may impose additional constraints on who is authorized to stage data.
```

Comment:
should probably be "Stage and read data", i.e. without the `/or` since both are included authorizations.

Reply:
The stage capability _allows_ the client to read the data, but such usage is not enforced. The capability can be used in an explicit call just to stage the data, while an actual read operation need not follow, for whatever reason.

Reply:
Text [adjusted](https://github.com/WLCG-AuthZ-WG/common-jwt-profile/pull/57/commits/0d4f8a4c6eaa0561ca5b6e2b8339200bbc353f4c) according to various comments in this PR.